Common/SecurityIdentifier.cs

```diff
@@ -751,6 +756,7 @@ public override bool Equals(object obj)
         public override string ToString()
         {
             var props = EncodeBase36(_properties);
+            props = props.IsNullOrEmpty() ? "0" : props;
```

Comment:
nit - `EncodeBase64` never returns null, faster to just check `props.Length == 0 ? "0" : props` -- also curious as to why this was needed and does it affect anything else? This _will_ affect serialization of `Symbol.Empty`, but not sure if we depend on that serialization anywhere. I can't think of a case where we send it down the pipe via ticker plant or history, etc, so this is likely safe, but wanted to bring attention to it.

Reply:
RE: try parse will fail -- interesting, I would have expected this to require us to update the parsing and not the serialization logic.